components/ip2location/common/constants.mjs

```diff
@@ -0,0 +1,68 @@
+const PACKAGE_OPTIONS = [
+  "WS1",
+  "WS2",
+  "WS3",
+  "WS4",
+  "WS5",
+  "WS6",
+  "WS7",
+  "WS8",
+  "WS9",
+  "WS10",
+  "WS11",
+  "WS12",
+  "WS13",
+  "WS14",
+  "WS15",
+  "WS16",
+  "WS17",
+  "WS18",
+  "WS19",
+  "WS20",
+  "WS21",
+  "WS22",
+  "WS23",
+  "WS24",
+  "WS25",
+];
+
+const LANGUAGE_OPTIONS = [
+    "ar",
+    "cs",
+    "da",
+    "de",
+    "en",
+    "es",
+    "et",
+    "fi",
+    "fr",
+    "ga",
+    "it",
+    "ja",
+    "ko",
+    "ms",
+    "nl",
+    "pt",
+    "ru",
+    "sv",
+    "tr",
+    "vi",
+    "zh-cn",
+    "zh-tw",
+  ];
+
+const ADDON_OPTIONS = [
+    "continent",
+    "country",
+    "region",
+    "city",
+    "geotargeting",
+    "country_groupings",
+    "time_zone_info",
+  ];
+
+export {

```

Comment:
We usually export with
```suggestion
export default {
```
And then import it with
```javascript
import constants from "../../common/constants.mjs";
```
Then you can either destructor `constants` or reference the `const` ifself from `default` with `constatns.PACKAGE_OPTIONS`